lib/Distribution/Opensuse/Tumbleweed.pm

```diff
@@ -213,4 +216,16 @@ sub get_dns_server_setup {
     return YaST::DNSServer::DNSServerSetupController->new();
 }
 
```

Comment:
let's write a comment to distinguish that we are going to use pages from this line of code something like:
`# Page Object Design only with Pages, not using Controllers` wdyt?